src/Rating.php

```diff
@@ -54,4 +74,34 @@ public function getRates($originPostalCode, $postalCode, $weight, array $options
         );
         return $response;
     }
+    
+    /**
+     * Helper function to extract the option codes.
+     *
+     * @param array $options
+     *  The options array.
+     *
+     * @return array
+     *  The list of options with the option-code.
+     */
+    private function getOptionCodes(array $options) {
```

Comment:
**OPTIONAL CHANGE REQUEST**

In general it's good to use `protected` instead of `private` visibility by default so that if I want to inherit a class this method will be still available in the child. Unless you have a specific reason to not make it available.